composer.json

```diff
@@ -17,13 +17,13 @@
         "php": ">=5.3.3",
         "ext-mbstring": "*",
         "behat/gherkin": "^4.6.0",
-        "behat/transliterator": "^1.2",
-        "symfony/console": "~2.7.40||^2.8.33||~3.3.15||^3.4.3||^4.0.3",
-        "symfony/config": "~2.3||~3.0||~4.0",
-        "symfony/dependency-injection": "~2.1||~3.0||~4.0",
+        "behat/transliterator": "dev-master#41fa9aca9838d139a646920670c1c8f7e829e751",
+        "symfony/console": "~2.7.40||^2.8.33||~3.3.15||^3.4.3||^4.0.3||^5.0",
```

Comment:
will symfony dependencies work with just this: `~4.0||~5.0`

Reply:
I guess, but not in PHP 7.4. Do you think I should remove the transliterator changes now?